### PR TITLE
When evicting containers, perform a normal remove first

### PR DESCRIPTION
**Pull request by mheon**
This ensures that containers that didn't require an evict will be dealt with normally, and we only break out evict for containers that refuse to be removed by normal means.

### CODE REVIEW

**Comment by mheon** [comment]
@vrothberg @baude @rhatdan PTAL

File: libpod/runtime_ctr.go [hunk 1]
```diff
@@ -576,11 +576,20 @@ func (r *Runtime) evictContainer(ctx context.Context, idOrName string, removeVol
...
 	if err != nil {
 		return "", errors.Wrapf(err, "Failed to find container %q in state", idOrName)
 	}
 
+	// Begin by trying a normal removal. Valid containers will be removed normally.
+	tmpCtr, err := r.state.Container(id)
+	if err == nil {
+		logrus.Infof("Container %s successfully retrieved from state, evict not required", c.ID())
+		// Assume force = true for the evict case
+		return r.removeContainer(ctx, tmpCtr, true, removeVolume, false)
```
[the remark this answers is not in the document]
Comment (mheon) [reply libpod/runtime_ctr.go]: Hm. If it fails and the container is still valid, that's a good idea.

[the remark this answers is not in the document]
Comment (mheon) [reply libpod/runtime_ctr.go]: Fixed

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Pull request merged**